operator/apis/loki/v1/lokistack_types.go

```diff
@@ -844,6 +950,14 @@ type LimitsTemplateSpec struct {
 	// +kubebuilder:validation:Optional
 	QueryLimits *QueryLimitSpec `json:"queries,omitempty"`
 
+	// OpenTelemetry to configure which resource, scope and log attributes
+	// to store as labels or structured metadata or drop them altogether
+	// for all tenants.
+	//
+	// +optional
+	// +kubebuilder:validation:Optional
+	OpenTelemetry *GlobalOTLPSpec `json:"openTelemetry,omitempty"`
```

Comment:
nit: I would be more specific with the name. `opentelemetry` can mean many things. `otlp` makes more sense.